### DISCUSSION TITLE
Can i using this homebrew to dump psn game licence?

### DISCUSSION THREAD

**Discussion by pftmclub** [bucanero/apollo-ps3#29] [Q&A] [answered]
My ps3 has been installed digital game from ps store and ready to play. The problems is i lost password on this user account. 
So i want to ask is this possible to dump my game license to *.rap file on my usb for backup purposes  ?

**Comment by bucanero** [comment, marked as answer]
if the content license is still installed on the PS3 (the user activation wasn't erased), then yes you can use Apollo to export the license to `.rap` on USB.

Just open Apollo, select the "user backup" menu, and check the "export licenses" options. The app will list all the available licenses in the account, and let you export to `.rap` .

**Comment by pftmclub** [comment]
> if the content license is still installed on the PS3 (the user activation wasn't erased), then yes you can use Apollo to export the license to `.rap` on USB.
> 
> Just open Apollo, select the "user backup" menu, and check the "export licenses" options. The app will list all the available licenses in the account, and let you export to `.rap` .

I know this is not related to topic, but is there anyway to dump the content not just license file without sign in the psn account ?

**Comment by bucanero** [comment]
> I know this is not related to topic, but is there anyway to dump the content not just license file without sign in the psn account ?

you could use a manager like Multiman or Irisman to copy folders from the `/dev_hdd0/game/` directory, but I don't know if that would work or not.
